recipes/sasview/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0d49040321cd7ada4df35c366e9759a05fae2c0f5082cf57f4eaa69f0ab16f04" %}  # openssl dgst -sha256 v5.0-beta.2.tar.gz
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: v5.0-beta.2.{{ file_ext }}
+  url: https://github.com/SasView/{{ name }}/archive/v5.0-beta.2.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - sasview_setup.patch
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
```

It looks like this has [entry_points](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#python-entry-points):
```suggestion
  skip: True  # [win or py2k]
  entry_points:
    - sasview = sas.qtgui.MainWindow.MainWindow:run_sasview
```

---

Yes, it has. What's the advantage of defining the entry points in the recipe while they already exist in the `setup.py` file?

---

Conda isn't aware of the entry points defined inside the `setup.py` file so this information has to be duplicated.

For a `noarch: python` package this information allows it to be built on one platform (i.e. Linux) and then installed on another (i.e. Windows) with the correct wrapper scripts around these entry points. This doesn't apply here but I'm not sure if there are any other uses that are applicable so it's safer to include it.

---

OK, let's try this, though after the installation the `sasview` executable/entry point is available, otherwise the [test command](https://github.com/conda-forge/staged-recipes/pull/7839/files#diff-c1893145bb30029f6fcf9e51475b6826R54) would fail.